components/ActionSheet/tests/unit/MDCActionSheetTest.m

```diff
@@ -457,4 +463,28 @@ - (void)testSettingBaseOverrideBaseElevationReturnsSetValue {
   XCTAssertEqualWithAccuracy(self.actionSheet.mdc_overrideBaseElevation, fakeElevation, 0.001);
 }
 
+- (void)testSetHeaderDividerColor {
```

Comment:
Please add a test that setting the `headerDividerColor` causes the bounds/frame to increase by exactly 1 point.

Reply:
It only changes the contentOffset of the tableView and I have added test for that.